requirements.txt

```diff
@@ -0,0 +1,35 @@
+appdirs==1.4.3
+asgiref==3.2.10
+CacheControl==0.12.6
+certifi==2019.11.28
+cffi==1.14.3
+chardet==3.0.4
+colorama==0.4.3
+contextlib2==0.6.0
+cryptography==2.9.2
+distlib==0.3.0
+distro==1.4.0
+Django==3.1.2
+django-cors-headers==3.5.0
+djangorestframework==3.12.1
+html5lib==1.0.1
+idna==2.8
+ipaddr==2.2.0
+jwt==1.0.0
```

Comment:
I had to do pyjwt in mine, does this work as well?

Reply:
Yeah just the jwt also works